docs/NSR.md

```diff
@@ -301,7 +298,7 @@ The CA MUST ensure personnel assigned to Trusted Roles that are authorized to ac
 
 ###### 2.2.1.2
 
-The CA MUST NOT allow group accounts or shared role credentials to authenticate to or access CA Infrastructure and/or Network Equipment.
+The CA SHOULD NOT allow group accounts or shared role credentials to authenticate to or access CA Infrastructure and/or Network Equipment. When group accounts or shared role credentials are used, the CA MUST implement compensating controls, such as strict access control and auditing of system access.
 
```

Comment:
Something like:
> When group accounts or shared role credentials are used, the CA MUST be able to attribute each usage to

* an approved activity; and
* an individual user or service account.